maintain-tests.mdx

```diff
@@ -0,0 +1,86 @@
+---
+title: Maintain your tests
+description: "How to fix your broken tests easily"
+icon: "traffic-cone"
+---
+
+## Why tests fail
+
+`Red` signals the test has failed. This is due to 3 reasons:
+
+1. **You have a bug in your app**
```

Comment:
I think we need to get away from the generic wording that you have a bug and rather start talking about the fact that you **introduced** a bug. Something that used to work (covered by the test) is now not working anymore (what we like to call a regression in dev/product terms). Maybe even build a bridge that this is easily spotted if you produce regular test reports (scheduling *wink*) because then you actually know at which point in the time the bug got introduced (not just that it broke!)

Reply:
too broad of a discussion for a PR comment. I go basic with the wording for docs, for the part of the audience who has no clue what regression is, docs have to be generic a bit. And to the point, I try to keep it shorter to keep people RTFM. Suggest something if you have a snappy version in mind